tests/ui/specialization/min_specialization/bad-const-wf-doesnt-specialize.rs

```diff
@@ -2,11 +2,11 @@
 
 // An impl that has an erroneous const substitution should not specialize one
 // that is well-formed.
-
+#[derive(Clone)]
```

Comment:
This is necessary because otherwise, since we no longer emit a "conflict implementation" error, we then will fail b/c both impls are not WF (since `S` doesn't impl `Clone`) :see_no_evil: